src/components/textInput/TextInputBase.tsx

```diff
@@ -392,6 +392,7 @@ export const TextInputBase = ({
           disableFullscreenUI={true}
           accessibilityState={{ disabled }}
           accessibilityLabel={accessibilityLabel ?? placeholder}
+          accessibilityHint={status === 'error' ? bottomMessage : ''}
```

Comment:
I think that assuming this as a generic rule may be lead to some inconsistencies... what about adding another prop to accessibility options and handle any kind of logic on the parent component?

Reply:
I completely agree! In this case this logic can be moved in the TextInputValidation component. Excluding from it the accessibilityHint prop from external props. What do you think about it?

Reply:
I would leave the problem to define the hint in the component/screen integrating the `TextInputValidation` avoiding to create a generic scope that may lead to wrong behaviours, maybe just define a `accessibilityHint` prop to be drilled down to the `TextInputBase` what do you think?

Reply:
It makes sense! I will implement it!